plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererUserModificationTracker.kt

```diff
@@ -57,6 +57,21 @@ data class AcceptedSuggestionEntry(
     val connection: ToolkitConnection?
 ) : UserModificationTrackingEntry
 
+data class CodeInsertionDiff(
+    val original: String,
+    val modified: String,
+    val diff: Double
+)
+
+fun CodeInsertionDiff?.percentage(): Double = when {
+    this == null -> 1.0
```

Comment:
why 1 and not 0

Reply:
yea so that's why i left this comment https://github.com/aws/aws-toolkit-jetbrains/pull/4770/files#r1706502340

Reply:
it's been there since 2019 maybe

Reply:
and i also feel 1 is not correct